course_access_groups/acl_backends.py

```diff
@@ -13,6 +13,7 @@
     GroupCourse,
     Membership,
 )
+from course_access_groups.security import is_active_staff_or_superuser
```

Comment:
Just carrying over my comment from PR #17 that I think this would be better to be named `permissions` instead of `security` for consistency with `edx-platform`